app/bundles/ReportBundle/Views/Report/export.html.twig

```diff
@@ -0,0 +1,78 @@
+{% extends 'MauticCoreBundle:Default:slim.html.twig' %}
+
+{% block pageTitle pageTitle %}
+
+{% block headerTitle report.name %}
+
+{% block mauticContent 'report' %}
+
+{% block _content %}
+  {#$showGraphsAboveTable = (true === !empty($report->getSettings()['showGraphsAboveTable']));#}
+  {%- set showGraphsAboveTable = report.settings.showGraphsAboveTable is defined -%}
```

Comment:
Looking at the comment of how it used to be and how it is now. I don't think this will fly. As [settings](https://github.com/mautic/mautic/blob/5.x/app/bundles/ReportBundle/Entity/Report.php#L78) is private. It should be:
```suggestion
  {#$showGraphsAboveTable = (true === !empty($report->getSettings()['showGraphsAboveTable']));#}
  {%- set showGraphsAboveTable = report.getSettings().showGraphsAboveTable is defined -%}
```
Right? Although the access to private properties is everywhere. Twig can access private properties?

Reply:
`report.settings` and `report.getSettings()` will return the same result. For reference, you can read the _Implementation_ section under https://twig.symfony.com/doc/3.x/templates.html#variables

Reply:
I see. Magic. Thanks!